test/letters/containers/Main.unit.spec.jsx

```diff
@@ -50,51 +51,57 @@ describe('<Main>', () => {
     expect(vdom).to.not.be.undefined;
   });
 
-  it('renders its children when letters are available', () => {
+  it('shows a loading spinner when awaiting response', () => {
     const tree = SkinDeep.shallowRender(<Main {...defaultProps}/>);
-    const childText = tree.subTree('span').text();
-    expect(childText).to.equal(testText);
+    expect(tree.subTree('LoadingIndicator')).to.not.be.false;
   });
 
-  it('shows a loading spinner when awaiting response', () => {
-    const props = _.merge({}, defaultProps, { lettersAvailability: awaitingResponse });
+  it('shows a loading spinner when letters is available but address is awaiting response', () => {
```

Comment:
This is probably not even necessary, but should we throw in another test to ensure that a loading spinner is displayed when address is available but letters are still pending?

Reply:
Yes I was thinking that too, I will add it!